plugin/better-whitespace.vim

```diff
@@ -65,6 +65,12 @@ call s:InitVariable('strip_max_file_size', 1000)
 " Disable verbosity by default
 call s:InitVariable('better_whitespace_verbosity', 0)
 
+" Bypass the aliases set for diff by default
+if has("win32") || has("win16")
+    call s:InitVariable('diff_binary', 'diff')
```

Comment:
Probably safer to use `diff.exe` here, to also guard against if someone has something like `diff.cmd` in their path as well.

I was also going to say this would handle the case if someone did `set shell=powershell.exe` since PowerShell has a built-in `diff` alias that would break this, but after some testing it appears having shell set to PowerShell mangles how the `system` command in Vim passes the auto-gen'd temp file from the input string anyway... So I guess that's just not a supported use case then 😕

Reply:
Good point. The goal here was to target the diff that is shipped with vim on windows, I think it is indeed called vim.exe. I’ll check and add that change.